src/components/Footer.svelte

```diff
@@ -0,0 +1,155 @@
+<script>
+  import content from '@contentful-entry/footer'
+
+  const linkColumns = Object.keys(content).reduce((columns, contentKey) => {
```

Comment:
Let's move what we can of this function to the Rollup plugin - it will be useful in the future for consolidation, too. I can do it later tonight, unless someone wants to take it now...

@ArkaneMoose @bholmesdev